### PR TITLE
docs(*): add doc on signed helm charts

### PR DESCRIPTION
**Pull request by vdice**
As well as self-hosted public key used to sign charts.

cc @slack @kmala

### CODE REVIEW

File: src/installing-workflow/index.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ $ helmc install workflow-v2.8.0               # injects resources into
 
     	$ helm install deis/workflow --version=v2.8.0 --namespace=deis -f <optional values file>  # injects resources into your cluster
 
+  See also our section on [Workflow chart provenance](workflow-helm-charts.md#chart-provenance)
+
```
Comment (kmala) [inline src/installing-workflow/index.md]: should we add it to our other install sections in providers?

Comment (vdice) [inline src/installing-workflow/index.md]: @kmala good point -- actually, hoping a follow-up PR to address https://github.com/deis/workflow/issues/570 will remove/condense all the other (duplicated) sections, so perhaps moot by then.

Comment (vdice) [inline src/installing-workflow/index.md]: @kmala updated

**Comment by vdice** [comment]
added the `needs manual testing` label as I'd like to see another reviewer run some of the key fetching/chart verification commands as presented in this doc PR

**Comment by mboersma** [comment]
I tested as described here with `--verify`. 👍

**Pull request merged**